lib/templates/.pipeline/lib/config.js

```diff
@@ -1,7 +1,6 @@
 'use strict'
 const options = require('pipeline-cli').Util.parseArguments()
 const { OpenShiftClient } = require('pipeline-cli')
-const { logger } = require('../../../logger')
```

Comment:
it's because this config.js file uses (upon Initializer) in the apps (`.pipeline/lib` folder) so this link won't be valid. It's valid in nr-pipeline-ext repo only